book/src/snapshot-controller.md

```diff
@@ -12,6 +12,7 @@ When Volume Snapshot is promoted to Beta in Kubernetes 1.17, the CSI external-sn
 
 Latest stable release | Branch | Min CSI Version | Max CSI Version | Container Image | [Min K8s Version](kubernetes-compatibility.md#minimum-version) | [Max K8s Version](kubernetes-compatibility.md#maximum-version) | [Recommended K8s Version](kubernetes-compatibility.md#recommended-version)
 --|--|--|--|--|--|--|--
+[external-snapshotter v3.0.0](https://github.com/kubernetes-csi/external-snapshotter/releases/tag/v3.0.0) | [release-3.0](https://github.com/kubernetes-csi/external-snapshotter/tree/release-3.0) | [v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | - | k8s.gcr.io/sig-storage/snapshot-controller:v3.0.0 | v1.17 | - | v1.17
```

Comment:
@xing-yang we'll want to add a new section/page for the validation webhook.

And maybe a top-level section for client libraries?

Reply:
Sure.  I can submit a PR for that